anisoap/representations/radial_basis.py

```diff
@@ -207,6 +229,16 @@ def orthonormalize_basis(self, features: TensorMap):
             l_2n_arr = l + 2 * n_arr
             # normalize all the GTOs by the appropriate prefactor first, since the overlap matrix is in terms of
             # normalized GTOs
+            # Ensure that each gto width is the cutoff_radius * sqrt(n) / nmax. If n < 1, then take n = 1.
```

Comment:
Also going to revert these changes too, so this PR strictly shows the manual radial.